JavaFXUserInterface/src/main/resources/css/revamp/assetTypeInfo.css

```diff
@@ -0,0 +1,182 @@
+.titleBar {
+    -fx-background-color: #FFFFFF;
+}
+
+.formLabel {
+    -fx-fill: #3A3A3A;
+    -fx-font-size: 14px;
+    -fx-font-family: 'Segoe UI';
+}
+
+#deleteBtn {
```

Comment:
There is a lot of duplication in the CSS files that could be moved to the same file.